hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -458,4 +458,11 @@ message TransferLeadershipRequestProto {
 }
 
 message TransferLeadershipResponseProto {
+}
+
+message DeletedBlocksTransactionInfo {
+    optional int64 txID = 1;
+    optional int64 containerID = 2;
+    repeated int64 localID = 3;
+    optional int32 count = 4;
```

Comment:
@Xushaohong , there is a DeletedBlocksTransaction defined in StorageContainerDatanodeProtocolProtos, which has the same fields as this one. Could we use that directly?

Reply:
As I discussed with Ethan above, it is a historical problem that the **DeletedBlocksTransaction**  in `ScmServerDatanodeHeartbeatProtocol.proto` is not in `hdds.proto` as a general PB, here if we want to reuse it we need to import `ScmServerDatanodeHeartbeatProtocol.proto` in `ScmAdminProtocol.proto` which may cause some possible compatibility issue in the future, and hence not a good option.